test_settings.py

```diff
@@ -41,7 +41,7 @@ def root(*args):
     root('super_csv', 'conf', 'locale'),
 ]
 
-ROOT_URLCONF = 'super_csv.urls'
+# ROOT_URLCONF = 'super_csv.urls'
```

Comment:
I tried checking out and running tests without commenting this out; is this still necessary?

Reply:
Indeed, it's not necessary. I'm not sure why my tests failed previously.